st2actions/st2actions/cmd/scheduler.py

```diff
@@ -109,7 +109,7 @@ def _run_scheduler():
             handler.shutdown()
             entrypoint.shutdown()
         except:
-            pass
+            LOG.exception('Unable to shutdown scheduler.')
```

Comment:
Please see comment in the old PR https://github.com/StackStorm/st2/pull/5158 to address concerns from reviewer.